src/libraries/System.Drawing.Common/tests/GdiplusTests.cs

```diff
@@ -7,7 +7,7 @@ namespace System.Drawing.Tests
 {
     public class GdiplusTests
     {
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOSX))]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOSX), nameof(PlatformDetection.IsDrawingSupported))]
```

Comment:
Can we instead disable for ARM64? I don't want to be in the case where the non arm mac's are refreshed and the libgdiplus dependency is messed up (not installed or something) and this test helps us catch that libgdiplus is there.

Reply:
sure

Reply:
done